templates/travis/.travis/release.py.j2

```diff
@@ -42,9 +43,12 @@ def validate_redmine_data(redmine_query_url, redmine_issues):
             stats[f"milestone_{milestone}"].append(issue)
         except ResourceAttrError:
             stats["without_milestone"].append(issue)
+        {%- endif %}
 
+    {%- if update_redmine %}
+    if milestone_id is not None:
```

Comment:
it is weird, the milestone should be set on line 42

Reply:
it should have a milestone_id, otherwise, the job for updating redmine will fail

Reply:
if there are no issues assigned to a milestone, it won't be set.

Reply:
the script will stop early if there is at least one issue not assigned to a milestone, it won't proceed further.

Reply:
so we need an if here:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/update_redmine.sh
to check it the commit message contains the redmine query:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/update_redmine.sh

Reply:
What is the case when there is no redmine query and validate method passes so it gets to the update_redmine.sh?
A release with no redmine issues is the only case I can think of.

Reply:
I believe this is the only case